src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -169,19 +170,22 @@ public void Http20ConnectionClosed()
         }
 
         [NonEvent]
+        [UnsupportedOSPlatform("browser")]
```

Comment:
Why decorate internal class? It should be just excluded for browser

Reply:
It is referenced in `HttpClient` and `HttpMessageInvoker` which are included in browser build and browser build fails with `'HttpTelemetry' does not exist in the current context`

Reply:
This class should have different implementation for browser anyway.

Could you start with empty stubs copied from 
https://github.com/dotnet/runtime/blob/05efa9be9ff185c1e7da939213683c6806126c80/src/libraries/System.Net.Http/tests/UnitTests/Fakes/HttpTelemetry.cs as `HttpTelemetry.Browser.cs` and keep only needed methods?

Reply:
Separated only the unsupported methods https://github.com/dotnet/runtime/blob/ed2b8444e33df90e7b723737796a1fa60622880f/src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/HttpTelemetry.Browser.cs#L8-L22